rs/nns/cmc/src/main.rs

```diff
@@ -729,6 +729,24 @@ fn get_principals_authorized_to_create_canisters_to_subnets_() {
     })
 }
 
+#[candid_method(query, rename = "get_default_subnets")]
+fn get_default_subnets() -> Vec<PrincipalId> {
```

Comment:
Looks good - can you add a test in nns/integration_tests/src/cycles_minting_canister.rs?

`test_cmc_create_canister_refunds()` is a good example of the current way we have been writing the state machine (versus using `state_machine_test_on_nns_subnet`)

Reply:
I added a pretty basic test, I think the use-case for this will be quite narrow at the moment so these tests should be enough

Reply:
Perfect, thank you!